projects/EWTeam/build.xml

```diff
@@ -88,7 +90,25 @@ No need for Javadoc - this is not an API.
                <isset property="repo.version"/>
             </not>
         </condition>
-	</target>
+    </target>
+    <target name="-getenv" if="exists.GITHUB_RUN_NUMBER">
+       <add operand1="${lastGITLABbuild}" operand2="${env.GITHUB_RUN_NUMBER}" result="repo.version0"/>
+<!--
+TODO consider switching back to this at 9.8.0/9.9build
+        <property name="repo.version0" value="${env.GITHUB_RUN_NUMBER}"/>
+-->
+        <condition property="version.add" value="~${env.GITHUB_REF_NAME}" else="">
+           <not>
+               <equals arg1="${env.GITHUB_REF_NAME}" arg2="master"/>
+           </not>
+        </condition>
+        <property name="repo.version" value="${repo.version0}${version.add}"/>
+
+    </target>
+
+    <target name="properties" depends="copy-pc2jar, -execgit, -getenv">
+		<available classname="org.eclipse.jdt.core.JDTCompilerAdapter" property="build.compiler" value="org.eclipse.jdt.core.JDTCompilerAdapter" />
+    </target>
```

Comment:
Again, it's EWTeam so....  But the addition of almost 20 lines of XML with no explanatory comments isn't very helpful to those who are asked to review (not to mention those who will look at it later...)